airflow/providers/slack/operators/slack.py

```diff
@@ -195,49 +195,50 @@ class SlackAPIFileOperator(SlackAPIOperator):
     :type content: str
     """
 
-    template_fields = ('channel', 'initial_comment', 'filename', 'filetype', 'content')
+    template_fields = ('channel', 'initial_comment', 'filetype', 'content')
     ui_color = '#44BEDF'
 
     def __init__(
         self,
         channel: str = '#general',
         initial_comment: str = 'No message has been set!',
-        filename: str = None,
+        file: str = None,
         filetype: str = None,
         content: str = None,
         **kwargs,
     ) -> None:
         self.method = 'files.upload'
         self.channel = channel
         self.initial_comment = initial_comment
-        self.filename = filename
+        self.file = file
```

Comment:
I think it's better to leave that as file_name. I see no point why we should change it to `file`

Reply:
reverted back to filename, idea was to keep it consistent with the actual slack API expected file.params, but I understand the breakage concern.